sorting/bubble_sort.c

```diff
@@ -1,95 +1,22 @@
-/**
- * @file
- * @brief [Bubble sort](https://en.wikipedia.org/wiki/Bubble_sort) algorithm
- * implementation
- */
-#include <assert.h>
-#include <stdbool.h>
 #include <stdio.h>
-#include <stdlib.h>
-#include <time.h>
 
-/**
- * Display elements of array
- * @param arr array to be display
- * @param n length of array
- */
-void display(const int *arr, int n)
-{
-    for (int i = 0; i < n; i++)
-    {
-        printf("%d ", arr[i]);
-    }
-    printf("\n");
-}
+int main(){
+#define SIZE 5   // defining a macro
+int arr[SIZE] = {3,6,2,7,1};   // taking a demo array
```

Comment:
```suggestion
int main()
{
    #define SIZE 5   // defining a macro
    int arr[SIZE] = {3,6,2,7,1};   // taking a demo array
```
Makes the code more readable